infrastructure-playbooks/rolling_update.yml

```diff
@@ -1,6 +1,10 @@
 ---
 # This playbook does a rolling update for all the Ceph services
-# Change the value of 'serial:' to adjust the number of server to be updated.
+#
+# The value of 'serial:' adjusts the number of server to be updated simultaneously.
+# We recommend a value of 1, which means hosts of a group (ie: monitor) will be
+# upgraded one by one. It is really crucial for the update process to happen
+# in a serialized fashion. DO NOT CHANGE THIS VALUE.
```

Comment:
if we are encouraging serialized behavior why is this an option? Is there a way to hard code it?